UPGRADE.md

```diff
@@ -10,12 +10,98 @@ PR: https://github.com/laravel/passport/pull/1734
 
 PHP 8.1 is now the minimum required version.
 
+### Minimum Laravel Version
+
+PR: https://github.com/laravel/passport/pull/1744
+
+Laravel 10.0 is now the minimum required version.
+
 ### OAuth2 Server
 
 PR: https://github.com/laravel/passport/pull/1734
 
 The `league/oauth2-server` Composer package which is utilized internally by Passport has been updated to 9.0, which adds additional types to method signatures. To ensure your application is compatible, you should review this package's complete [changelog](https://github.com/thephpleague/oauth2-server/blob/master/CHANGELOG.md#900---released-2024-05-13). 
 
+### Client Secret
+
+PR: https://github.com/laravel/passport/pull/1744
+
+Passport now always hashes clients' secret, you may call the following command if you need to hash your clients' secrets:
+
+    php artisan passport:hash
+
+### Client ID
+
+PR: https://github.com/laravel/passport/pull/1744
+
+Passport now uses UUID to identify clients. If you were already using client with UUIDs you don't have to make any changes, but if you were not using client with UUIDs you must change the type of client ID columns to `char(36)`. Your previous client IDs won't change and you will get UUID for newly created clients from now on:
+
+```php
+Schema::table('oauth_clients', function (Blueprint $table) {
+    $table->char('id', 36)->change();
```

Comment:
Maybe primary?

Reply:
It is already primary, we just [change the type](https://laravel.com/docs/11.x/migrations#:~:text=does%20not%20change%20the%20indexes%20of%20the%20column) here.